lectures/markov_chains_II.md

```diff
@@ -174,17 +174,17 @@ distribution, then, for all $x \in S$,
 ```{math}
 :label: llnfmc0
 
-\frac{1}{m} \sum_{t = 1}^m \mathbf{1}\{X_t = x\}  \to \psi^*(x)
+\frac{1}{m} \sum_{t = 1}^m \mathbb{1}\{X_t = x\}  \to \psi^*(x)
     \quad \text{as } m \to \infty
 ```
 
 ````
 
 Here
 
-* $\{X_t\}$ is a Markov chain with stochastic matrix $P$ and initial.
-  distribution $\psi_0$
-* $\mathbb{1} \{X_t = x\} = 1$ if $X_t = x$ and zero otherwise.
+* $\{X_t\}$ is a Markov chain with stochastic matrix $P$ and initial distribution $\psi_0$
+
+* $\mathbf{1} \{X_t = x\} = 1$ if $X_t = x$ and zero otherwise.
```

Comment:
@longye-tian , I think this should be mathbb.  (We should make all of them `\mathbb{1}`.)

Reply:
Yes, thank you for pointing this out! I have just updated this issue.